telemetry/definitions/commonDefinitions.json

```diff
@@ -1598,6 +1598,19 @@
                 }
             ]
         },
+        {
+            "name": "amazonq_modifySourceFolder",
+            "description": "User modified source folder",
+            "metadata": [
+                {
+                    "type": "amazonqConversationId"
+                },
+                {
+                    "type": "credentialStartUrl",
+                    "required": false
+                }
```

Comment:
Since these are the only fields, this indicates that this metric is literally just a boolean. Why can't it be a field on an existing metric?